AUTHORS

```diff
@@ -68,6 +68,9 @@ ScummVM Team
     BBVS:
        Benjamin Haisch
 
+    Buried:
+       Matthew Hoops
+
```

Comment:
Will do, and will mark Matthew as (retired) as per sev's other comment to ensure he is not bothered about issues.